test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeModule.cs

```diff
@@ -130,31 +130,71 @@ public Task UpdateDesiredPropertiesAsync(object patch, CancellationToken token)
                 this.Id);
         }
 
-        public Task WaitForReportedPropertyUpdatesAsync(object expectedPatch, CancellationToken token)
```

Comment:
nit: expected could be more descriptive

Reply:
Given the name of the method, it seems clear to me that you'd call it when you want to wait until the reported properties on this module match the `expected` object you passed in.